webapp/src/main/java/com/box/l10n/mojito/quartz/QuartzPollableTaskScheduler.java

```diff
@@ -75,7 +79,12 @@ public <I, O> PollableFuture<O> scheduleJobWithCustomTimeout(
    * @param <O>
    * @return
    */
-  public <I, O> PollableFuture<O> scheduleJob(QuartzJobInfo<I, O> quartzJobInfo) {
+  public <I, O> PollableFuture<O> scheduleJob(
+      QuartzJobInfo<I, O> quartzJobInfo, String schedulerName) {
```

Comment:
discussed offline but for the record, let's pass the scheduler name into the quartzJobInfo